estuary_updater/handlers/base.py

```diff
@@ -136,8 +136,11 @@ def get_or_create_build(self, identifier, original_nvr=None, force_container_lab
             # Certain Koji API endpoints omit the *_ts values but have the *_time values, so that's
             # why the *_time values are used
             if build_info[time_key]:
-                build_params[time_key] = datetime.strptime(
-                    build_info[time_key], '%Y-%m-%d %H:%M:%S.%f')
+                ts_format = r'%Y-%m-%d %H:%M:%S'
+                if len(build_info[time_key].rsplit('.', 1)) == 2:
+                    # If there are microseconds, go ahead and parse that too
```

Comment:
... because, why not?

Reply:
Well some include them and some don't, so I figured I'd include them when they do for consistency rather than just strip them off.